### PR TITLE
#776 USB Import Wizard

### LINKED ISSUE #776: ✨ Ballot Import USB Wizard
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Suggestion

As an admin when I upload ballots I want to see a screen that reads: "Insert USB Drive".  Then when I insert my USB drive with ballots it will automatically start importing them.

### Possible Implementation

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by lprichar**
### Issue

Fixes #776 

### Description
Adds an alternative screen for importing from USB devices.  This screen only shows up when you're running locally from Python (e.g. `make egui`).  It is cancellable by hitting the subtle "x" button in the top right which will return to the old ballot upload screen.  On load it looks for a USB device with an artifacts directory.  If it doesn't find one, it prompts the user to insert a drive and allows them to click "Scan Drives" to look again.

image

When it finds a candidate it displays information from the drive:

image

When the user clicks "Import" it shows a status description during upload:

image

On success it notifies if there were any warnings or issues and allows the user to import again or exit out:

image

### Testing

Try using the wizard as described above.

### CODE REVIEW

File: src/electionguard_gui/web/components/admin/upload-ballots-legacy-component.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
+import RouterService from "../../services/router-service.js";
```
Comment (lprichar) [inline src/electionguard_gui/web/components/admin/upload-ballots-legacy-component.js:1]: This file is unchanged, github failed to detect the rename

File: src/electionguard_gui/web/components/admin/upload-ballots-wizard-component.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      this.duplicateCount = 0;
+      this.status = null;
+      this.loading = false;
+      await this.scanDrives();
+    },
+    scanDrives: async function () {
+      const result = await eel.scan_drives()();
+      if (!result.success) {
+        console.error(result.message);
+        // todo: show error
```
Comment (lprichar) [inline src/electionguard_gui/web/components/admin/upload-ballots-wizard-component.js:58]: oops, I'll do this now

**Review by SteveMaier-IRT — APPROVED** [review]
lgtm

**Review by SteveMaier-IRT — APPROVED** [review]
looks good.

**Pull request merged**